### PR TITLE
ya-runtime-vm adding optional -device param to vmrt

### PR DESCRIPTION
**Pull request by pwalski**
Resolves: https://github.com/golemfactory/golem-internal/issues/647

Todo:
- [x] sync with ya-runtime-vm master
- [x] Remove pci devices list related code (it will be done by self-test-img)
- [x] Implement adding `!exp:gpu` capability based on `self-test-img` output
- [x] Move changes in qemu `Dockerfile` to dedicated repository where it will be build together with gpu dedicated vmlinuz
- [x] Create CI for `/golemfactory/ya-runtime-vm-gpu` which takes binary of configurable `ya-runtime-vm` release, builds `qemu`, and `vmlinuz` and publishes it as an `ya-runtimve-vm-gpu` release (it will still require installation script which will generate runtime dedscriptor)
- [x] Implement adding optional `--device` Qemu param configurable by runtime-vm cmdline `--pci-device` param in runtime descriptor `extra-args` field.

### CODE REVIEW

**Comment by pwalski** [comment]
`qemu` build moved to https://github.com/golemfactory/ya-runtime-vm-gpu

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -68,6 +69,24 @@ pub struct Cli {
...
+    /// Test process timeout (in sec)
+    #[structopt(long, env = "YA_RUNTIME_VM_TEST_TIMEOUT", default_value = "10")]
+    test_timeout: u64,
+    /// Number of logical CPU cores for test process
+    #[structopt(long, env = "YA_RUNTIME_VM_TEST_CPU_CORES", default_value = "1")]
+    test_cpu_cores: usize,
+    ///  Amount of RAM for test process [GiB]
+    #[structopt(long, env = "YA_RUNTIME_VM_TEST_MEM_GIB", default_value = "0.125")]
+    test_mem_gib: f64,
+}
```
[the remark this answers is not in the document]
Comment (nieznanysprawiciel) [reply runtime/src/lib.rs:84]: Later in many places in code this would allow to avoid multiplying code around these values

**Comment by pwalski** [comment]
In case of approve do not merge it - need to check last time everything on GPU host before merge.

File: runtime/src/lib.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (pwalski) [reply runtime/src/lib.rs]: Moved params to single struct which gets flatten by `structopt`.

**Pull request merged**